src/pages/home/HomePage.js

```diff
@@ -62,7 +62,7 @@ class HomePage extends PureComponent {
         PersonalDetails.fetch();
         PersonalDetails.fetchTimezone();
         getBetas();
-        fetchAllReports(true, false, true);
+        fetchAllReports(true, true, true);
```

Comment:
I think this is the only case where we were passing `false` before so we can probably undo all the logic in `fetchAllReports()` that checks to see if we should also load actions?

Reply:
https://github.com/Expensify/Expensify.cash/blob/f6ca82ca208a7d627b62f08054d72c05f6b3e6f3/src/libs/actions/Report.js#L541-L546